dev/com.ibm.websphere.appserver.features/visibility/public/mpMetrics-5.0/io.openliberty.mpMetrics-5.0.feature

```diff
@@ -16,10 +16,11 @@ Subsystem-Name: MicroProfile Metrics 5.0
   com.ibm.websphere.appserver.anno-2.0, \
   com.ibm.websphere.appserver.monitor-1.0
 -bundles=com.ibm.ws.microprofile.metrics.common, \
- io.openliberty.io.smallrye.common.jakarta, \
```

Comment:
so io.openliberty.io.smallrye.common.jakarta isn't needed?

Reply:
Nope. As per slack conversation. This was added initially to follow in line with other MP features that pulled SmallRye implementations .

However, this is not needed for `mpMetrics-5.0` to run.